Formula/srt.rb

```diff
@@ -0,0 +1,28 @@
+class Srt < Formula
+  desc "Secure, Reliable, Transport"
+  homepage "http://srtalliance.org"
+  url "https://github.com/Haivision/srt/archive/v1.2.0.tar.gz"
+  sha256 "ca22506cfafa18fb1f9c6dd5ceebe5e56c1b205105b2dd3736701aeea3f4145a"
+
+  head do
+    url "https://github.com/Haivision/srt.git"
+  end
+
+  depends_on "pkg-config" => :build
+  depends_on "cmake" => :build
+  depends_on "openssl"
+
+  def install
+    openssl = Formula["openssl"].opt_prefix
+
+    system "./configure", "--prefix=#{prefix}",
```

Comment:
Doesn't it find the `openssl` libraries on its own, without the two `--with-openssl`?

Reply:
Without these arguments, it tries to use system-wide openssl, instead of brew's openssl.
The problem is that SRT won't work appropriately if it is linked with the openssl which Mac OS provides. Therefore, SRT requests to set openssl directories manually.